go.mod

```diff
@@ -5,15 +5,15 @@ go 1.12
 require (
 	cloud.google.com/go v0.43.0
 	github.com/crossplaneio/crossplane v0.3.1-0.20191023221351-518648b051cd
-	github.com/crossplaneio/crossplane-runtime v0.0.0-20191023215652-0f37bea5496e
+	github.com/crossplaneio/crossplane-runtime v0.0.0-20191025043010-78072ef19dc5
 	github.com/google/go-cmp v0.3.0
 	github.com/googleapis/gax-go v1.0.3
 	github.com/onsi/gomega v1.5.0
 	github.com/pkg/errors v0.8.1
 	golang.org/x/exp v0.0.0-20190731235908-ec7cb31e5a56 // indirect
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
 	golang.org/x/tools v0.0.0-20190806215303-88ddfcebc769 // indirect
-	google.golang.org/api v0.8.0
+	google.golang.org/api v0.11.0
```

Comment:
Is this required or just a side-effect?

Reply:
I have no idea why that bumped. As far as I know I ran only `go get github.com/crossplaneio/crossplane-runtime`, which I would not expect to affect this. 🤔

Reply:
I wonder if it is somehow nested as a dependency in a `crossplane-runtime` dependency.

Reply:
Not sure how it snuck in, but I unstaged the commit, repeated the process, and now it's gone. 🤷‍♂